datastore/additional_data/sources/geo_lookup.py

```diff
@@ -0,0 +1,144 @@
+import csv
+import hashlib
+import io
+import json
+import zipfile
+from datetime import datetime
+
+import requests
+from django.forms.models import model_to_dict
+
```

Comment:
Not sure you're actually using model_to_dict?

Reply:
now removed